utils/post.js

```diff
@@ -163,15 +164,16 @@ const modifyAnonymousAndBlockPost = async (
   return feedWithAnonymous;
 };
 
-const modifyAnonimityPost = async (item, isBlurredPost = true) => {
+const modifyAnonimityPost = async (item, isBlurredPost = true, anonymUserId) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the function `modifyAnonimityPost` has a new parameter `anonymUserId` added at the end. This can break existing calls to this function that are not aware of this change. It's better to add new parameters in a way that doesn't break existing functionality, for example by using an options object.

```diff
- const modifyAnonimityPost = async (item, isBlurredPost = true, anonymUserId) => {
+ const modifyAnonimityPost = async (item, {isBlurredPost = true, anonymUserId}) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->